sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ConsumeValueTaskCorrectly.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System.Threading.Tasks;
+
+namespace Tests.Diagnostics
+{
+    class ValueTaskProvider
+    {
+        public ValueTask<int> ReadAsync() => new ValueTask<int>();
+        public void Foo(int x) { }
+
+    }
+
+    class Tests
+    {
+        // see https://devblogs.microsoft.com/dotnet/understanding-the-whys-whats-and-whens-of-valuetask/
+        // and https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1?view=netstandard-2.1
+
+        public async void Foo1(ValueTaskProvider stream)
+        {
+            var valueTask = stream.ReadAsync();
+            var once = await valueTask; // Noncompliant { "Refactor this 'ValueTask' usage to consume it only once." }
```

Comment:
the syntax for this should actually be `{{Refactor this 'ValueTask' usage to consume it only once.}}
` , so this doesn't check the message...